### ISSUE TITLE
"What can be left here for recycling" should use recycling:glass in the Czech Republic

### ISSUE THREAD

**Issue by kudlav** [westnordost/StreetComplete#2123]
Answering "Glass bottles and jars" overrides existing value "recycling:glass=yes" which is in Czech Republic prefered, so this quest destroys properly mapped data.

According to czech general recycling guide we can recycle:
Coloured glass (used for wine, beer, etc.), smashed glass panes of doors or windows, bottles and jars, broken beverage glass.
See https://jaktridit.cz/en/sorting/sorting-waste-properly---coloured-containers/glass

You can see related issue on Osmose project, after discussion in talk-cz the check was disabled for the Czech Republic: https://github.com/osm-fr/osmose-backend/issues/691

**How to Reproduce**
Example node that will be overridden:
https://www.openstreetmap.org/node/2815619731

**Versions affected**
Android 10
StreetComplete 23.0

**Comment by westnordost** [comment]
There are two quests: One that asks whether only bottles can be recycled here, or any type of glass if `recycling:glass = yes and !recycling:glass_bottles`.
The other, like you said, only offers to select "glass bottles and jars".

So I would propose to change the following:
1. disable the `DetermineRecyclingGlass` quest type for CZ
2. **Either** for CZ offer both options "Any type of glass" / "glass bottles and jars" **or** replace the "glass bottles and jars" option with an "Any type of glass", whatever makes more sense. Do **all** containers accept all kinds of glass? It sounds a bit odd that it would accept window glass, but the slot to throw things in is bottle-shaped.

**Comment by kudlav** [comment]
Hi, I would prefer dialog for Glass similar to plastics. Typical containers are for any type of glass but there can be smaller one only for bottles as you said.

![Screenshot_20200930-133406_StreetComplete](https://user-images.githubusercontent.com/11516259/94680468-10c43e80-0322-11eb-8a19-f68ba4c86357.png)

**Comment by westnordost** [comment]
Is it written on the container itself what is allowed and what is not? Because if it is not, users may give the wrong answer because they don't know better

**Comment by kudlav** [comment]
Yes it is or there is just "Glass". But you can put any kind of glass except mirrors and glass from cars so it rather depends on size of container and size of hole.

These two would be mapped as "Any type of glass":
![Container big](https://www.jaktridit.cz/uploads/images/Gallery/fotogalerie/sklo/kontejnery/Deleny-kontejner-na-sklo.jpg)
![Container opened](http://files.zalepsifirmy.webnode.cz/200000110-a4199a513e/kontejner_sklo_barevne.jpg)

Maybe this is "Only for bottles" container, but I don't know what's written on the top, I will explore that later. But mapping everything as "Any type of glass" would be less harmful than mapping everything as "Only for bottles".
![Container opened](https://servis.idnes.cz/fbimg.aspx?foto=RZE741830_tridenyodpad.jpg&c=A180619_091340_praha-zpravy_rsr)

**Comment by westnordost** [comment]
Ok, I'll take care of it

**Comment by westnordost** [comment]
My plan would be to show a dialog similar to plastic only for Czech Republic for now. If later it appears that all kinds of glass can be recycled in other countries, they will be added to the list for which the behavior is activated.

**Comment by FloEdelmann** [comment]
@westnordost Why only for specific countries though? The `DetermineRecyclingGlass` quest is shown to all users anyway, so why not merge the quests for all users? It's just one click more.

**Comment by westnordost** [comment]
Because you won't find any normal container in Germany for example that accepts window glass so it does not make sense to display that option - it will only lead to mistaggings from people who don't know better and didn't read what's printed on the container.

**Comment by FloEdelmann** [comment]
Okay, but then the `DetermineRecyclingGlass` also doesn't make sense in Germany, does it?

**Comment by westnordost** [comment]
It does, because many containers are incorrectly tagged as glass=yes

**Comment by westnordost** [comment]
I decided to not show an option (any glass / glass bottle and jars only) for the same reason why the option is not shown for Germany: Posing it as an option in CZ would be a source of mistakes. If there is really a rare container that does not accept window glass, the user can leave a note. The same for Germany and other countries where the user finds a container that **does** accept window glass.

**Comment by westnordost** [comment]
![Screenshot_1602596596](https://user-images.githubusercontent.com/4661658/95868699-d10e4580-0d6a-11eb-9671-d372124baf27.png)
